src/test/java/lotto/WinningNumberTest.java

```diff
@@ -17,49 +17,24 @@ public class WinningNumberTest {
     @Test
     @DisplayName("보너스 번호 매치 테스트")
     public void bonusNumberMatch(){
+        List<LottoNumber> lottoNumbers = LottoTicket.valueOf(List.of(3,5,7,9,13,11));
         WinningNumbers winningNumbers = new WinningNumbers(List.of(3,5,7,9,6,10),11);
-        assertThat(winningNumbers.bonusNumberMatch(
-                List.of(
-                        LottoNumber.of(3)
-                        ,LottoNumber.of(5)
-                        , LottoNumber.of(7)
-                        , LottoNumber.of(9)
-                        , LottoNumber.of(13)
-                        , LottoNumber.of(11)
-                )
-            )
-        ).isTrue();
+        assertThat(winningNumbers.bonusNumberMatch(lottoNumbers)).isTrue();
     }
 
     @Test
     @DisplayName("보너스 번호 매치 실패 테스트")
     public void bonusNumberFailMatch(){
+        List<LottoNumber> lottoNumbers = LottoTicket.valueOf(List.of(3,5,7,9,13,11));
```

Comment:
```suggestion
        List<LottoNumber> lottoNumbers = LottoTicket.valueOf(3, 5, 7, 9, 13, 11);
```
가변인자 적용해 위와 같이 구현하는 것은 어떨까?